<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwdiget</class>
 <widget class="QWidget" name="mainwdiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ聊天</string>
  </property>
  <widget class="QPushButton" name="pb_img">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resImages.qrc">
     <normaloff>:/images/tx/30.png</normaloff>:/images/tx/30.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="l_nick">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_feeling">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>个人签名</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>331</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_space">
    <attribute name="title">
     <string>空间</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_friend">
    <attribute name="title">
     <string>朋友</string>
    </attribute>
    <widget class="QListWidget" name="lw_friendlist">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_massage">
    <attribute name="title">
     <string>消息</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>323</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_menu">
      <property name="text">
       <string> 菜单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_func1">
      <property name="text">
       <string> 功能1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_func2">
      <property name="text">
       <string>功能2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_func3">
      <property name="text">
       <string> 功能3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resImages.qrc"/>
 </resources>
 <connections/>
</ui>
